templates/commands/goldentest/record.go

```diff
@@ -37,17 +37,17 @@ type RecordCommand struct {
 }
 
 func (c *RecordCommand) Desc() string {
-	return "record the template rendering result to golden tests"
+	return "record the template rendering result to golden tests(capture the anticipated outcome akin to expected output in unit test)"
```

Comment:
Same here and below with the space before the parenthesis

Reply:
done